src/Estimators/tests/RobotDynamicsEstimator/RobotDynamicsEstimationTest.cpp

```diff
@@ -293,9 +426,13 @@ TEST_CASE("RobotDynamicsEstimator Test")
         REQUIRE((output.tau_F - dataset.expectedTauF.row(sample).transpose()).isZero(0.1));
         REQUIRE((output.tau_m - dataset.expectedTaum.row(sample).transpose()).isZero(0.1));
         REQUIRE(((output.tau_m - output.tau_F) - dataset.expectedTauj.row(sample).transpose()).isZero(0.1));
-        for (auto const& [key, value] : dataset.fts)
+        for (int idx = 0; idx < sensors["ft"].size(); idx++)
+        {
+            REQUIRE((output.ftWrenches[sensors["ft"][idx].ukfSensorName] - dataset.fts[sensors["ft"][idx].sensorName].row(sample).transpose()).isZero(0.1));
+        }
+        for (int idx = 0; idx < sensors["contact"].size(); idx++)
         {
-            REQUIRE((output.ftWrenches[key] - value.row(sample).transpose()).isZero(0.1));
+            REQUIRE((output.contactWrenches[sensors["contact"][idx].ukfSensorName]).isZero(5.0));
```

Comment:
5.0 as tolerance is a lot https://eigen.tuxfamily.org/dox/classEigen_1_1DenseBase.html#af36014ec300f53a65083057ed4e89822